doc/user/spark.md

```diff
@@ -1,4 +1,115 @@
-## Spark Support
+# Get Started with DAOS Hadoop Filesystem
+Here, we describe the steps required to build and deploy the DAOS Hadoop filesystem, and the configurations to access
+DAOS in Spark. We assume DAOS servers and agents have already been deployed in the environment, otherwise they can be
+deployed by following the [DAOS installation guide](https://daos-stack.github.io/admin/installation/).
 
-Spark integration with libdfs is under development and is scheduled for DAOS
-v1.2.
+## Build DAOS Hadoop Filesystem
+The DAOS DFS Java API and Hadoop filesystem implementation have been merged into DAOS repository. Below are the steps
+to build the java jar files for the DFS Java API and DAOS Hadoop filesystem. These jar files are required when running
+Spark. You can ignore this section if you already have the pre-built jars.
+
+
+    $ git clone https://github.com/daos-stack/daos.git
+    $ cd daos
+    $ git checkout <desired branch or commit>
+    ## assume DAOS is built and installed to <daos_install> directory
+    $ cd src/client/java
+    $ mvn clean package -DskipITs -Ddaos.install.path=<daos_install>
+
+After build, the package daos-java-<version>-assemble.tgz will be available under distribution/target.
+
+## Deploy DAOS Hadoop Filesystem Jars
+After unzipping daos-java-<version>-assemble.tgz, you’ll get the following files.
```

Comment:
You need to add a blank line before bullets otherwise mkdocs does not recognize it.

Reply:
I just checked the rendering of spark.md in the "sparkdoc" branch in github. it works as expected. Is there any misunderstanding?

Reply:
we use mkdocs and i am afraid that it has a different interpreter. Anyway, i can fix that when updating the online site.